website/cue/reference/releases/0.22.0.cue

```diff
@@ -0,0 +1,767 @@
+package metadata
+
+releases: "0.22.0": {
+	date:     "2022-05-24"
+	codename: ""
+
+	whats_next: [
+		{
+			title: "Removal of legacy buffers"
+			description: """
+				With this release, [we've switched the default for disk buffers to the new `v2`
+				implementation](/highlights/2022-04-06-disk-buffer-v2-stable). Tthis means if you set `type
+				= "disk"` you will get the new buffer implementation. In a future release, we will remove the legacy
+				disk buffers.
```

Comment:
I know we link to the highlight article, but would it make sense to add a one-liner ensuring people there'll be no loss of data?

Reply:
Also, minor nit, but it is a bit confusing that we're talking about "switching defaults in this release", when this blurb is in the "what's next" section. It might confuse people to think whatever is written here applies to the next release, not this release.

Reply:
Perhaps simply adding "With this release (0.22.0)" solves the ambiguity.